src/components/ListCard/listcards.js

```diff
@@ -1,52 +1,66 @@
 import React from "react";
 import "./listcards.css";
+import { useNavigate } from "react-router-dom";
 
 const personalLists = [
-  ["Check out their Favourites"], 
-  ["Check out their Wishlist"],
-]; //To be changed after demo 
+	["Check out their Favourites"],
+	["Check out their Wishlist"],
+]; //To be changed after demo
 
 const images = ["/images/noma.jpeg", "/images/durum.jpeg"];
 
 function ListCards() {
-  const titles = ["💖 Favourites", "🌠 Wishlist"];
-
-  return (
-    <div>
-      {titles.map((title, index) => {
-        const truncatedText = personalLists[index].slice(0, 5);
-
-        if (personalLists[index].length > 5) {
-          truncatedText[5] = truncatedText.slice(5) + " and more";
-        }
-
-        return (
-          <Card
-            key={index}
-            title={title}
-            text={truncatedText.join(", ")}
-            imageSource={images[index]}
-          />
-        );
-      })}
-    </div>
-  );
+	const titles = ["💖 Favourites", "🌠 Wishlist"];
+
+	return (
+		<div>
+			{titles.map((title, index) => {
+				const truncatedText = personalLists[index].slice(0, 5);
+
+				if (personalLists[index].length > 5) {
+					truncatedText[5] = truncatedText.slice(5) + " and more";
+				}
+
+				return (
+					<Card
+						key={index}
+						title={title}
+						text={truncatedText.join(", ")}
+						imageSource={images[index]}
```

Comment:
does not seem to return image for favorite restaurant. i chose retaurant that has an image in feeed, added to favs, and when checked favs, the image is not rendered.